roles/check_local_time/README.md

```diff
@@ -0,0 +1,30 @@
+# cluster_config
+
+Role check_local_time can be used to:
+- check if local time in provided time zone is located within required time interval
```

Comment:
Maybe add extra line or two:

This can be used to ensure certain tasks/playbooks are executed only during allowed time interval.
For example, HyperCore version upgrade might be allowed only outside of business hours.

Reply:
Added extra lines